impress-org/givewp
# Changed: Form Grid block styles

## Description

This PR updates the styles of the Form Grid block to better match the design in [Figma](https://www.figma.com/file/mF8Hn0p6Oc3yIQ2WpHHHH0/Form-Grid-Updated-Designs?node-id=42%3A648).

Also, the issue with overriding the Donate button text is resolved. Closes #6406.

Additional updates:

- Progress bar values now work correctly
- New Inspector Control layout
- Goals based on donor amount, donation amount and percentages display in the progress bar section
- Tags display on form grid in two alternate ways depending on featured images

## Affects

Form Grid block layout

## Visuals



https://user-images.githubusercontent.com/75056371/173704700-30c31bcc-ec53-4f2d-9cbb-cdd8b41e864d.mov



## Testing Instructions


-Create several forms with different donation goal options set.
-Process Donations on all forms
-Add excerpt messages, tags and featured images to some or all the forms
-View the Form Grid block
-Test all inspector controls to verify they work properly.
-Verify new design against the figma details


## Pre-review Checklist
